addons/portal/data/mail_template_data.xml

```diff
@@ -34,12 +34,12 @@
             <table border="0" cellpadding="0" cellspacing="0" width="590" style="min-width: 590px; background-color: white; padding: 0px 8px 0px 8px; border-collapse:separate;">
                 <tr><td valign="top" style="font-size: 13px;">
                     <div>
-                        Dear <t t-out="object.user_id.name or ''">Marc Demo</t>,<br/> <br/>
-                        Welcome to <t t-out="object.user_id.company_id.name">YourCompany</t>'s Portal!<br/><br/>
+                        Dear <t t-out="object.partner_id.name or ''">Marc Demo</t>,<br/> <br/>
```

Comment:
There's a similar line further up, in the header. It probably needs updating too